test/e2e/network/ingress.go

```diff
@@ -818,7 +828,9 @@ var _ = common.SIGDescribe("Ingress API", func() {
 
 		expectFinalizer := func(ing *networkingv1.Ingress, msg string) {
 			framework.ExpectNotEqual(ing.DeletionTimestamp, nil, fmt.Sprintf("expected deletionTimestamp, got nil on step: %q, ingress: %+v", msg, ing))
-			framework.ExpectEqual(len(ing.Finalizers) > 0, true, fmt.Sprintf("expected finalizers on ingress, got none on step: %q, ingress: %+v", msg, ing))
+			if len(ing.Finalizers) <= 0 {
```

Comment:
this can not be negative AFAIK, `==`

Reply:
Yeah, length can't be negative. Sorry for that. I had updated it.